tests/run-make/glibc-symbols-x86_64-unknown-linux-gnu/rmake.rs

```diff
@@ -0,0 +1,110 @@
+// Check that the compiler toolchain (rustc) that we distribute is not using newer glibc
+// symbols than a specified minimum.
+// This test should only be executed on an extracted dist archive or in a dist-* CI job.
+
+//@ only-dist
+//@ only-x86_64-unknown-linux-gnu
+//@ ignore-cross-compile
+
+use std::path::{Path, PathBuf};
+
+use run_make_support::{cmd, llvm_objdump, regex, rustc_path};
+
+fn main() {
+    // This is the maximum glibc version *supported* by the x86_64-unknown-linux-gnu target.
```

Comment:
Question: actually, did you mean *minimum*?

Reply:
Oh this is kinda worded strangely, this is the *maximum* **permitted** glibc version for symbols that appear in the dist artifacts, but that corresponds to the *minimum* supported glibc version we promise for `x86_64-unknown-linux-gnu`, right?

Reply:
Yeah, exactly, it's the classical min/max confusion thing. We promise that our minimal requirement is glibc 2.17, which means that we have to support >= glibc 2.17, which means that we have to use *at most* 2.17 symbols.

Reply:
Right, that makes sense. Maybe worth slightly adjusting the comment / variable name here.